README.md

```diff
@@ -138,6 +138,47 @@ Requirements:
 > Note: this project includes a [Maven Wrapper](https://maven.apache.org/wrapper/) that can be used instead of a system installed Maven.
 > Use it by replacing `mvn` by `./mvnw` on Linux and `mvnw.cmd` on Windows in the commands below.
 
+### Install JDK on M1
```

Comment:
I wrote:

```
"how to install JDK on M1" is not suitable to document on the Get Started page. Imagine that you don't tell users how to buy a computer. It's far away from Pulsar topics. Users can resolve it with Google, StackOverflow, etc.
```

in https://github.com/apache/pulsar/issues/17409#issuecomment-1236839244. This stands for README.md also. I can image users have multiple approach to do this and we don't have to use forty lines for it. If someone asked, we resolve the discuss or issue. And followers should be able to search or we simply answer by throwing the link.

Reply:
For example, I install JDK on M1 simply with `sdk install java` without all the troubleshooting here and it works well.

Reply:
This issue happened to me and 5 other engineers I know of just as an example. 
Our approach, especially on getting started should be giving you everything you need to get started - in the README case, it's building the product. M1 is notorious to have many quirks. Anything we can do to help will be greatly appreciated.

Just mentioning a few people recently running into those issues @geomagilles @teabot @nahguam just to name a few

Bear in mind, that they can ignore this recommendation. Most newcomers will appreciate it as much as I did.